modules/op-running-a-pipeline.adoc

```diff
@@ -19,6 +19,7 @@ $ tkn pipeline start build-and-deploy \
     -p deployment-name=pipelines-vote-api \
     -p git-url=https://github.com/openshift/pipelines-vote-api.git \
     -p IMAGE=image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/pipelines-vote-api
```

Comment:
`-p IMAGE=image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/pipelines-vote-api \`, `\` is missing